### ISSUE TITLE
Why translation parameters are not included in this mano model?

### ISSUE THREAD

**Issue by lishiyu005** [lixiny/manotorch#19]
Dear author,

Thanks for the great work. I try the code(https://github.com/lixiny/manotorch/blob/master/scripts/test_compatibility.ipynb) and find an issue with the compatibility with Omid's and Yana's Mano model.

In Omid's MANO model( https://github.com/otaheri/MANO)
omid_mano_model(
    global_orient=random_pose[:,:3],
    hand_pose=random_pose[:,3:],
    transl = None,
    betas=random_shape, 
    return_verts=True,
    return_tips = True
There are global orientation (N, 3), hand pose: (N, 45) , betas: (N, 10) shape coefficients, transl: (N, 3) translation vector used in the Mano model.

In Yana's manopth (https://github.com/hassony2/manopth)
def forward(self,
                th_pose_coeffs,
                th_betas=torch.zeros(1),
                th_trans=torch.zeros(1),
                root_palm=torch.Tensor([0]),
                share_betas=torch.Tensor([0]),
                ):
        """
        Args:
        th_trans (Tensor (batch_size x ncomps)): if provided, applies trans to joints and vertices
        th_betas (Tensor (batch_size x 10)): if provided, uses given shape parameters for hand shape
        else centers on root joint (9th joint)
        root_palm: return palm as hand root instead of wrist
        """
 th_trans is also provided as the translation parameters of the hand.

Why translation of the hands is not included in this repo? Will this be added in the future?

Best regards

**Comment by lixiny** [comment]
Translation do not affect how LBS works. 
The translation  can be directly applied by adding the 3-dim vector to all MANO joints or vertices
Maybe take a look at the code in manopth:[code](https://github.com/hassony2/manopth/blob/4f1dcad1201ff1bfca6e065a85f0e3456e1aa32b/manopth/manolayer.py#L268C1-L269C56)

and in Omid's MANO:[code](https://github.com/otaheri/MANO/blob/5869ab059c1bf31cc724f57eaf93e041135e8960/mano/model.py#L353C16-L354C62)

I don’t think this will be added to this repo.
That’s exactly how SMPL was designed — translation lives in the world coordinate system, not inside the model.

**Comment by lishiyu005** [comment]
Thank you
